lib/message.rb

```diff
@@ -0,0 +1,22 @@
+require 'twilio-ruby'
+
+class Message
+  attr_reader :account_sid, :auth_token,
+              :client, :from, :to
+
+  def initialize(client = Twilio::REST::Client.new('ACbad1cc1fe45037b01a7fd207fb7f270c', '2223f6422dca48314885d82d94b3d005'))
```

Comment:
Data like this should be stored in environment variables or a config file that is not committed - to protect the data. 
Please look at removing this data from GitHub and using environment variables to store and access it locally.